modules/customizing-the-jenkins-image-stream-tag.adoc

```diff
@@ -0,0 +1,35 @@
+:_content-type: PROCEDURE
+
+[id="customizing-the-jenkins-image-stream-tag_{context}"]
+= Customizing the Jenkins image stream tag
+
+To control how the Jenkins image is upgraded, you can override the default upgrade behavior by setting the image stream tag value.
+
+The default upgrade behavior is determined by the `"name": "scheduled-upgrade-redeploy"` value in the `jenkins-rhel.json` image stream file. This image stream tag value automatically redeploys the latest version of the Jenkins image when it is released.
+
+To override that default value, you change the value of the  `JENKINS_IMAGE_STREAM_TAG` in the `jenkins-ephemeral.json` Jenkins template. For example, you replace the `2` in `"value": "jenkins:2"` with one of the following the image stream tags:
+
+* `ocp-upgrade-redeploy` updates your Jenkins image when you upgrade {product-title}, you use this image stream tag in your Jenkins deployment configuration.
+* `user-maintained-upgrade-redeploy` requires you to manually redeploy Jenkins after you upgrade {product-title}.
```

Comment:
```suggestion
* `user-maintained-upgrade-redeploy` requires you to manually redeploy, by running `oc import-image` per the instructions at <add link>, your Jenkins instance.
```

Reply:
Updated this by adding the full command. Unfortunately, we can't use cross-reference links in our doc modules, so I can put a simple link to the procedure here.